packages/components/src/components/domainproperties/dataset/actions.ts

```diff
@@ -92,3 +152,41 @@ export function getHelpTip(fieldName: string): string {
     }
     return helpTip;
 }
+
+export function getDatasetProperties(datasetId?: number) {
+    return new Promise((resolve, reject) => {
+        Ajax.request({
+            url: ActionURL.buildURL('study', 'GetDataset'),
+            method: 'GET',
+            params: { datasetId },
+            scope: this,
```

Comment:
I don't think you need this scope param, doesn't look like it is used in the success or failure.